automation/run-tests.sh

```diff
@@ -1,8 +1,7 @@
-#!/bin/sh -ex
+#!/bin/bash -ex
 
 EXEC_PATH=$(dirname "$(realpath "$0")")
 PROJECT_PATH="$(dirname $EXEC_PATH)"
-: ${DOCKER_IMAGE:=nmstate/centos7-nmstate-dev}
 EXPORT_DIR="$PWD/exported-artifacts"
 CONT_EXPORT_DIR="/exported-artifacts"
 if [[ $DOCKER_IMAGE == *"fedora"* ]];then
```

Comment:
This check now needs to happen after the docker image command-line option was parsed.

Reply:
This is handled at line 163 after the `getopt()`.

Reply:
Line 163 sets the `DOCKER_IMAGE` variable but line 7 already checks the value of this variable to determine the python site path command. I do not see how this works correctly.

Reply:
it seems to me that this works in Travis only because DOCKER_IMAGE is set in travis.yml

Reply:
Oh, I noticed that, will fix it. Thanks.

Reply:
Done.